dialog/file_test.go

```diff
@@ -282,3 +283,29 @@ func TestFileFilters(t *testing.T) {
 	}
 	assert.Equal(t, 4, count)
 }
+
+func TestFileFavorites(t *testing.T) {
+	win := test.NewWindow(widget.NewLabel("Content"))
+
+	dlg := NewFileOpen(func(reader fyne.URIReadCloser, err error) {
+		assert.Nil(t, err)
+		assert.Nil(t, reader)
+	}, win)
+
+	dlg.Show()
+
+	favorites := dlg.dialog.loadFavorites()
+	places := dlg.dialog.loadPlaces()
+	assert.Equal(t, 3+len(places), len(favorites))
+
+	curDir := ""
+	for _, f := range favorites {
+		btn := f.(*widget.Button)
+		t.Log(fmt.Sprint("Changing from ", curDir, " to ", btn.Text))
```

Comment:
We don't normally print debug in tests - only relevant info is when a test fails

Reply:
I think this info is relevant to know why the test fails. There can be other reasons for why the test fails because I don't check against the exact directory name.

Reply:
I suppose my comment only makes sense in combination with the one below.

Reply:
I removed the log